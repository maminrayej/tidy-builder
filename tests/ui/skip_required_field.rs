#[derive(tidy_builder::Builder)]
struct MyStruct {
    #[builder(skip)]
    set_later_opt: Option<usize>,

    #[builder(skip)]
    #[builder(default = 0)]
    set_later_def: usize,

    #[builder(skip)]
    req1: usize,
}

fn main() {}
