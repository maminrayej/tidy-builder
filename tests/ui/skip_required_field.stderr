error: Cannot skip a required field
  --> tests/ui/skip_required_field.rs:10:5
   |
10 | /     #[builder(skip)]
11 | |     req1: usize,
   | |_______________^
